EnglishFilter/sections/specific.txt

```diff
@@ -958,6 +958,7 @@ techably.com##.sidebar a[href^="https://affiliate.namecheap.com/"]
 sciencedaily.com##.mobile-top-rectangle
 sciencedaily.com##.mobile-middle-rectangle
 sciencedaily.com##.mobile-bottom-rectangle
+faucetpay.io##center > a[href*="advert"]
```

Comment:
Exchange to something like this `faucetpay.io##a[href^="https://faucetpay.io/advertise/"][target="_blank"]`